reana_job_controller/kubernetes_job_manager.py

```diff
@@ -316,6 +322,53 @@ def _add_krb5_init_container(self, secrets_volume_mount):
         self.job['spec']['template']['spec']['initContainers'].append(
             krb5_container)
 
+    def _add_proxy_init_container(self, secrets_volume_mount):
+        """Add  sidecar container for a job."""
+        ticket_cache_volume = {
+            'name': 'proxy-cache',
+            'emptyDir': {}
+        }
+        volume_mounts = [
+            {
+                'name': ticket_cache_volume['name'],
+                'mountPath': current_app.config['PROXY_CERT_CACHE_LOCATION']
+            }
+        ]
+
+        proxy_pass = os.environ.get('VOMSPROXY_PASS')
```

Comment:
Even if we are inside the function that creates the `voms_proxy` side card, we could use the `voms_` prefix wherever we call the variable `proxy_xxx`, for more clarity, what do you think? i.e. `voms_proxy_pass`.

Reply:
Absolutely agree, that's a relic from when the field was just called `proxy`